docs/source/developer/llvm_timings.rst

```diff
@@ -0,0 +1,105 @@
+.. _developer-llvm-timings:
+
+====================
+Notes on timing LLVM
+====================
+
+
+Getting LLVM Pass Timings
+-------------------------
+
+The dispatcher stores LLVM pass timings in the metadata under the
```

Comment:
```suggestion
The dispatcher stores LLVM pass timings in the dispatcher object metadata under the
```
and reference to this: https://numba.readthedocs.io/en/stable/reference/jit-compilation.html#Dispatcher.get_metadata